src/server/rest/v1/validator/AuthValidator.ts

```diff
@@ -16,7 +16,14 @@ export default class AuthValidator extends SchemaValidator {
   private authEula: Schema = JSON.parse(fs.readFileSync(`${global.appRoot}/assets/server/rest/v1/schemas/auth/auth-eula.json`, 'utf8'));
 
   private constructor() {
-    super('AuthValidator');
+    super('AuthValidator', {
```

Comment:
There is no extra param in other validator, let the default one in the super class.